nixos/modules/services/x11/hardware/libinput.nix

```diff
@@ -185,14 +180,63 @@ in {
           Option "DragLockButtons" "L1 B1 L2 B2"
         '';
         description = ''
-          Additional options for libinput touchpad driver. See
+          Additional options for libinput ${deviceType} driver. See
           <citerefentry><refentrytitle>libinput</refentrytitle><manvolnum>4</manvolnum></citerefentry>
           for available options.";
         '';
       };
-
     };
 
+    mkX11ConfigForDevice = deviceType: matchIs: ''
+        Identifier "libinput ${deviceType} configuration"
+        MatchDriver "libinput"
+        MatchIs${matchIs} "${xorgBool true}"
```

Comment:
Well, my reasoning to use the function is because X.org has many ways to set something to true (I think 1 and yes also works). So if someone wants to changes this later for some reason they will only change one place (`xorgBool`).